package.json

```diff
@@ -31,31 +31,31 @@
     "zone.js": "~0.13.0"
   },
   "devDependencies": {
-    "@angular-builders/jest": "15.0.0",
-    "@angular-devkit/build-angular": "^15.2.1",
-    "@angular-eslint/builder": "15.2.1",
-    "@angular-eslint/eslint-plugin": "15.2.1",
-    "@angular-eslint/eslint-plugin-template": "15.2.1",
-    "@angular-eslint/schematics": "15.2.1",
-    "@angular-eslint/template-parser": "15.2.1",
-    "@angular/animations": "^15.2.1",
-    "@angular/cli": "^15.2.1",
-    "@angular/compiler": "^15.2.1",
-    "@angular/compiler-cli": "^15.2.1",
-    "@angular/language-service": "^15.2.1",
-    "@angular/router": "^15.2.1",
+    "@angular-builders/jest": "16.0.0",
+    "@angular-devkit/build-angular": "^16.0.5",
+    "@angular-eslint/builder": "16.0.3",
+    "@angular-eslint/eslint-plugin": "16.0.3",
+    "@angular-eslint/eslint-plugin-template": "16.0.3",
+    "@angular-eslint/schematics": "16.0.3",
+    "@angular-eslint/template-parser": "16.0.3",
+    "@angular/animations": "^16.0.5",
+    "@angular/cli": "^16.0.5",
+    "@angular/compiler": "^16.0.5",
+    "@angular/compiler-cli": "^16.0.5",
+    "@angular/language-service": "^16.0.5",
+    "@angular/router": "^16.0.5",
     "@compodoc/compodoc": "^1.1.19",
-    "@ngneat/spectator": "^14.0.0",
+    "@ngneat/spectator": "^15.0.1",
     "@types/lodash-es": "^4.17.6",
     "@types/node": "^18.16.16",
     "@types/webpack-env": "^1.18.1",
-    "@typescript-eslint/eslint-plugin": "^5.59.6",
-    "@typescript-eslint/parser": "^5.59.8",
+    "@typescript-eslint/eslint-plugin": "^5.59.2",
+    "@typescript-eslint/parser": "^5.59.2",
     "@commitlint/cli": "^17.6.1",
     "@commitlint/config-conventional": "^17.6.5",
     "commitizen": "^4.2.5",
     "cz-conventional-changelog": "^3.3.0",
-    "eslint": "^8.42.0",
+    "eslint": "^8.39.0",
```

Comment:
a number of these eslint changes (above too) are downgrades - intentional?

Reply:
Yeah, this was modified by ng upgrade. Probably @angular-eslint forced ng to downgrade eslint version